### DISCUSSION TITLE
linux-modules-extra-6.2.5-surface missing

### DISCUSSION THREAD

**Discussion by Malecord** [linux-surface/linux-surface#1092] [Q&A] [answered]
Today for some reason wifi disappeared from my surface studio.

lshw -C network

shows that network is "UNCLAIMED"

Checking "linux-modules-extra" tells me that no "linux-modules-extra-6.2.5-surface" exists.

Any recommended way to fix that? Is there a place where I can download it?

**Comment by qzed** [comment, marked as answer]
We don't provide such a package, the modules are in the kernel package itself. Is there any kernel module that you're missing or that you expect to be there but is not?

**Reply by Malecord** [reply]
Hello Qzed,

I thought I was missing that module. That would have been the undesired result of an update gone wrong while I was on the go on train commute.

However today wifi is back on line so I guess whatever I did last time to repair it worked. I only had to restart once more.

Sorry for the noise.

**Reply by Malecord** [reply]
I got a further update on this issue.

It ust turned out that if I switch on my surface with the dock station attached the wifi module is not loaded and I remain without internet (since the dockstation has no wan port). Disconnecting the dock station doesn't work.

By switching on the laptop with the dock station disconnected, everything is initialized as expected. I can then connect the dock station and the wifi module remains on.
